### ISSUE TITLE
Add COPYING file to toplevel

### ISSUE THREAD

**Issue by xtian-kc** [ecb-org/ecb#7]
Add a COPYING file carrying the license information ECB is ppublished under (GPL v2+ currently) to the toplevel source directory.

**Comment by ware** [comment]
@xtian-kc, the copyright is easy to find.  That said, I'll add this for the next release.

**Comment by ware** [comment]
@xtian-kc, if you have any comments, please make them in pull request #25.

**Comment by shamefulCake1** [comment]
@ware @tec-leo this issue can be closed.

**Comment by ware** [comment]
> [@ware](https://github.com/ware) [@tec-leo](https://github.com/tec-leo) this issue can be closed.

You are completely correct!  BTW, thank you @shamefulCake1 for evaluating if all these issues should still be open or not.  It is appreciated.  I owe you a beverage of your choice!
